lib/views/home_content/feed/post_card/post_card.dart

```diff
@@ -9,7 +10,7 @@ import "package:proxima/views/navigation/routes.dart";
 /// This widget is used to display the post card in the home feed.
 /// It contains the post title, description, votes, comments
 /// and the user (profile picture and username).
```

Comment:
```suggestion
/// and the user (profile picture and username).
/// On click, the corresponding challenge (if it exists), is marked as completed.
```
You could update the documentation now that we can complete challenges :)